### DISCUSSION TITLE
Multitenancy in Directus

### DISCUSSION THREAD

**Discussion by bhagya1101** [directus/directus#16925] [Archive]
How do I perform Multitenancy in directus??
I want a super admin who can manage all other admins and each admin can manage a group of users.
Any help will be appreciated.

**Comment by danielband** [comment]
Currently there is no such thing as a super admin. You might be able to setup multiple Directus instances. Then you you can create administrator and user accounts for each instance.

**Reply by bhagya1101** [reply]
Thanks a lot.
